### PR TITLE
Update iwcd_short path and padding type

### PR DESCRIPTION
**Pull request by lbidulka**
Updated outdated index file path for iwcd_short.yaml
Updated outdated padding config option for iwcd_cnn_short.yaml

### CODE REVIEW

**Comment by lbidulka** [comment]
Updated engine_classifier.py to check for existing optimizer setup. Now it works if just running testing, which has no need for optimizer.

File: watchmal/engine/engine_classifier.py [hunk 1]
```diff
@@ -525,7 +525,7 @@ def restore_state_from_file(self, weight_file):
             self.model_accs.load_state_dict(checkpoint['state_dict'])
             
             # if optim is provided, load the state of the optim
-            if hasattr(self, 'optimizer'):
+            if getattr(self, 'optimizer') is not None:
```
Comment (nickwp) [inline watchmal/engine/engine_classifier.py:528]: I think this can just be
`if self.optimizer is not None:`
which reads a bit nicer. Could you check?

Comment (nickwp) [reply watchmal/engine/engine_classifier.py:528]: Maybe my version would fail if there is no `optimizer` attribute, but then the original code would have worked.

**Review by nickwp — APPROVED** [review]
Looks good to me now.
